lib/miq_expression.rb

```diff
@@ -657,8 +657,14 @@ def self.operands2rubyvalue(operator, ops, context_type)
     end
   end
 
+  def self.attribute?(val)
+    Field.parse(val).valid?
+  rescue LoadError, StandardError
+    false
+  end
+
   def self.quote(val, typ)
-    if Field.is_field?(val)
+    if attribute?(val)
```

Comment:
@yrudman isn't method already covered by this [method in MiqExpression::Field ](https://github.com/lpichler/manageiq/blob/8b30a0f755b85472b1b31c19d0eb2e8c030b9987/lib/miq_expression/field.rb#L20)

Reply:
@lpichler `Field.is_field` checks only if model is correct and do not check attribute. It would return `true` on "Vm-hello".

Reply:
I see ok and in that case, isn't better to have such method in MiqExpression::Field rather ? 

thanks!

Reply:
Ohh, I see, in issue description i sad "does not represent existing model", it should say "existing attribute from existing model", will clarify it in issue description ....